### PR TITLE
Todo list -ready to review

### PR DESCRIPTION
**Pull request by ahmetbozaci**
- Create add function for add todo list.
- Create delete function for delete todo item.
- Create drag drop function for change order todo item.

### CODE REVIEW

File: src/deleteTodo.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const deleteCompletedTodoItem = (tasks) => {
+  const clearAll = document.getElementsByClassName('clearAll');
+  clearAll[0].addEventListener('click', () => {
+    function finish(todo) {
+      return !todo.completed === true;
+    }
+    tasks = tasks.filter(finish);
+    for (let i = 0; i < tasks.length; i += 1) {
+      tasks[i].index = i;
+    }
```
Comment (ntzwilly) [inline src/deleteTodo.js:13]: This code uses an old version of for loop. You can make it look better by using forEach loop like this: 

forEach((task, i) => task.index = i)

File: src/deleteTodo.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const deleteOnlyOne = (tasks) => {
+  const trashIcon = document.querySelectorAll('.fa-trash-alt');
+
+  trashIcon.forEach((icon) => icon.addEventListener('click', () => {
+    const todoItem = icon.parentNode;
+    tasks.splice(todoItem.id, 1);
+    for (let i = 0; i < tasks.length; i += 1) {
+      tasks[i].index = i;
+    }
```
Comment (ntzwilly) [inline src/deleteTodo.js:27]: Here also, you can use forEach

File: src/addToStorage.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+const addToStorage = (tasks) => {
+  localStorage.setItem('items', JSON.stringify(tasks));
+};
```
Comment (ntzwilly) [inline src/addToStorage.js:3]: - [ ] As the body of addToStorage contains one single line, you can remove the braces to make the code shorter

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  ? JSON.parse(localStorage.getItem('items'))
+  : [];
+
+addToStorage(tasks);
+const data = JSON.parse(localStorage.getItem('items'));
+
+const createList = (todoItem) => {
+  if (!todoItem) {
+    return;
+  }
```
Comment (ntzwilly) [inline src/index.js:19]: You can shorten the if statement by removing the braces

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.classList.add('pointer');
+  checkbox.checked = todoItem.completed;
+
+  const descriptionSpan = document.createElement('span');
+  descriptionSpan.classList.add('text');
+
+  if (todoItem.completed) {
+    descriptionSpan.classList.add('check');
+  }
```
Comment (ntzwilly) [inline src/index.js:37]: You can remove the braces and add the code inside the if statement just after the closing paranthesis

**Review by ntzwilly — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 

## Hi, @ahmetbozaci

Your project looks good.  However, you can still improve your code by following JavaScript best practices.

*Please check my comments below for more detail*. 

## Cheers and Happy Coding :wave: :wave: :wave:

File: src/deleteTodo.js (hunk 1 above)
Comment (ahmetbozaci) [reply src/deleteTodo.js]: ✅  Thank you. I change it.

File: src/deleteTodo.js (hunk 2 above)
Comment (ahmetbozaci) [reply src/deleteTodo.js]: ✅  Thank you. I change it.

**Pull request closed without merging**